requirements/docs.txt

```diff
@@ -1,4 +1,5 @@
 Django>=2.2,<5.0
+sphinxcontrib-django
```

Comment:
We can pin this to a specific version if necessary but I don't really see the need